packages/extension/cypress/e2e/wallet_management.cy.ts

```diff
@@ -481,7 +481,7 @@ describe('Edit wallet', () => {
 
     // Lock the extension
     cy.get('button[aria-label="close"]').click();
-    cy.contains('button', 'Wallets').parent().children().eq(2).click();
+    cy.contains('button', 'Wallets').parent().children().eq(3).click();
```

Comment:
Why has this changed? Some comments for bellow.

Reply:
"NFTs" button has been added to the menu at position 2